internal/app/wwctl/overlay/imprt/main_test.go

```diff
@@ -34,7 +33,7 @@ func Test_List(t *testing.T) {
 		t.FailNow()
 	}
 
-	file, err := ioutil.TempFile(tmpdir, "file")
+	file, err := os.CreateTemp(tmpdir, "file")
```

Comment:
can we also add test cases for the verification of overlay names?
to check if it is the format of `__RUNTIME__**` and `__SYSTEM__**`.